metadoc-core/src/main/scala/metadoc/IndexLookup.scala

```diff
@@ -31,13 +32,13 @@ object IndexLookup {
 
   def findSymbol(
       offset: Int,
-      attrs: Attributes,
+      attrs: Document,
```

Comment:
doc? also, a bunch of other parameters called attrs.

Reply:
Good catch! I'm too excited to see things compile/work.